packages/rancher-sachet/charts/Chart.yaml

```diff
@@ -7,5 +7,5 @@ apiVersion: v2
 name: rancher-sachet
 description: A Helm chart for Sachet based on the upstream https://github.com/messagebird/sachet
 type: application
-version: 1.0.1
```

Comment:
no need to change this version since it's already overridden by the package.yaml version

Reply:
fixed.